tests/MMS/spatial/advection/runtest

```diff
@@ -93,7 +99,9 @@ for opts, label, sym, exp_ord in options:
     order = check_scaling.get_order(dx, error_2)
     print("Convergence order = {:f} ({:f} at small spacing)".format(*order))
 
-    if check_scaling.check_order(error_2, exp_ord):
+    # Loose tolerance because upwind-3rd order would otherwise fail at
+    # low resolution, when it is very good over the rest of the domain
+    if check_scaling.check_order(error_2, exp_ord, tolerance=3.e-1):
```

Comment:
Instead of loosening the tolerance for all the tests, could we just exclude the lowest nx values when testing upwind-3rd order? i.e. put in a conditional like
```
    if "3rd order upwind" in label:
        this_nxlist = nxlist[2:]
    else:
        this_nxlist = nxlist
    for nx in this_nxlist:
        ....
```

Reply:
I did think about this, but I'm not keen on special casing where we can avoid it. We could just drop the lowest point entirely?

Reply:
I'd prefer that to a much looser tolerance.